routes/dao_coin_exchange.go

```diff
@@ -176,8 +311,45 @@ func (fes *APIServer) GetTransactorDAOCoinLimitOrders(ww http.ResponseWriter, re
 		)
 		return
 	}
+	var buyingCoinPkid *lib.PKID
+	if requestData.BuyingCoinPublicKeyBase58Check != "" {
+		if IsDesoPkid(requestData.BuyingCoinPublicKeyBase58Check) {
```

Comment:
We could probably just simplify to setting buyingCoin to zero PKID and then just have an if !IsDesoPkid condition. Also maybe we refactor this into a helper function so that the logic of getting a PKID from a public key base 58 check that may be "deso" is all in one place and easily reused everywhere.